TestSendCore/Program.cs

```diff
@@ -86,7 +86,7 @@ private static IServiceProvider ConfigureServices()
                 options.SetMinimumLogLevel(LogLevel.Debug);
                 options.AddConsole();
 
-                var logstashAddress = Environment.GetEnvironmentVariable("LOGSTASH_ADDRESS");
+                var logstashAddress = "http://127.0.0.1:9090/"; Environment.GetEnvironmentVariable("LOGSTASH_ADDRESS");
```

Comment:
This seems like a bug?

Reply:
It's just a standalone test tool in addition to the unit tests. I don't think this change really matters, it's not included in the package.